### PR TITLE
[16.0][MIG] storage_backend_sftp: Migration to 16.0

### PR DESCRIPTION
**Pull request by santostelmo**
Migration of `storage_backend_sftp` to 16.0

ref: SOM-123

### CODE REVIEW

File: storage_backend_sftp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Storage Backend SFTP",
+    "summary": "Implement SFTP Storage",
+    "version": "16.0.1.0.2",
```
Comment (simahawk) [inline storage_backend_sftp/__manifest__.py]:
```suggestion
    "version": "16.0.1.0.0",
```

**Review by simahawk — CHANGES_REQUESTED** [review]
please rewrite the commit msg as `[MIG] storage_backend_sftp: Migration to 16.0`

**Comment by santostelmo** [comment]
> please rewrite the commit msg as `[MIG] storage_backend_sftp: Migration to 16.0`

done

**Review by leemannd — APPROVED** [review]

**Comment by bealdav** [comment]
@simahawk finally message commit rename with your expectations. Final review ?

**Comment by MiquelRForgeFlow** [comment]
Please cherry-pick https://github.com/OCA/storage/commit/cd5eddae1e7279674bd9ee0d20a1d11bac5ef03c, which was merged in v15.

**Comment by lmignon** [comment]
A rewrite of the storage_backend_* is proposed as a single addon supporting a wider list of platform through the use of the fsscpec python lib. You can take a look here https://github.com/OCA/storage/pull/252 and https://github.com/OCA/storage/pull/251

**Comment by gurneyalex** [comment]
@MiquelRForgeFlow The fix you are mentioning is on a different addon (storage_backend_ftp)

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**